packages/api/test/util/SimpleSigner.js

```diff
@@ -0,0 +1,13 @@
+
+export default class SimpleSigner{
```

Comment:
I would suggest renaming this one to `SingleAccountSigner` or just change it slightly to take in a keyring instance and then doing `keyring.getPair(options.from)` (Which is what I assumed the `SimpleSigner` to be, but maybe that is rather a `KeyringSigner` - the above suggested naming I would probably apply here anyway, just to be explicit)

Reply:
Just a reminder that this class is only supposed to be used in test. I don't think we need to ship a implementation of Signer with api. (From your comments I feel you think this is a exported class.)